templates/index.html

```diff
@@ -50,12 +58,16 @@ <h2 class="u-no-margin--bottom"><strong>A broad portfolio</strong></h2>
         <div class="row">
           <div class="col-6">
             <h3 class="p-heading--2">One stable platform <br>for everything open source</h3>
-            <p>Build with confidence using the world's favourite Linux operating system. <br>Ubuntu delivers long-term supported releases every two years.</p>
+            <p>Build with confidence using the world's favourite Linux operating system. <br>Ubuntu delivers long-term
+              supported releases every two years.</p>
```

Comment:
Same here, maybe for the `<br>` there can be a new line, but `supported releases` in a new line, doesn't make a lot of sense

Reply:
it doesn't change the layout, it's just matter of readability